app/ui/browser/model/index.js

```diff
@@ -25,6 +25,21 @@ export const Pages = Immutable.Record({
 }, 'Pages');
 
 /**
+ * The metadata parsed from a page for a Page record.
+ */
+export const PageMeta = Immutable.Record({
+  description: undefined,
+  icon_url: undefined,
+  image_url: undefined,
+  title: undefined,
+  type: undefined,
+  url: undefined,
+  player_url: undefined,
+  player_stream_url: undefined,
+  player_stream_content_type: undefined,
```

Comment:
Not quite sure what you're thinking of how this would work with Immutable and how we manage state. I'll just remove the player_* fields from now here